controllers/serverController.js

```diff
@@ -58,13 +59,15 @@ router.get('/:id', (req, res) => {
   });
 });
 
+/* TODO: re-add delete endpoint once security becomes a thing
 router.put('/:id', checkAccessKey, (req, res) => {
```

Comment:
isn't this what `checkAccessKey` is for?

Reply:
The access key is a public variable that is exposed, https://skybunk-auth-production.herokuapp.com/users. It offers no security other than obscurity.

Reply:
These are salted/hashed, in the same way user passwords are. We probably shouldn't be returning it in the response but from a security perspective there's nothing you can do with that value without knowing the salt & the original value

https://crackstation.net/hashing-security.htm

Reply:
I agree. The access key shouldn't be recoverable from the salted hash. However we still shouldn't be returning it in the response.
Realistically nothing is using these endpoints yet, so either commenting or adding the `checkAccessKey` would work. My vote is for the `checkAccessKey` path.

Reply:
True, I didn't realize it was salted hash. We can add checkAccessKey to everything, and then remove the post endpoint for now (I don't know the best way to secure the post endpoint)